utils/das_data_sorter.py

```diff
@@ -122,20 +122,22 @@ def average_data(self, data:pd.Series) -> pd.Series:
 
         return average_data_array
     
-    # def gps_data(self, data:pd.Series) -> pd.Series:
-        '''currently doesn't work as intended'''
+    def gps_data(self, data:pd.Series) -> pd.Series:
+        '''Returns the data array of the time intervals which the GPS was turned on. 
         
-    #     new_gps_data = []
-    #     for index_array in self.indexes:
-    #         if 1 is in data[index_array]:
-    #             new_gps_data.append(1)
-    #         else:
-    #             new_gps_data.append(0)
+        0 for when it was turned off at that point, 1 for when turned on.
+        '''
+        new_gps_data = []
+        for index_array in self.indexes:
+            if 1 in data[index_array].values:
+                new_gps_data.append(1)
+            else:
+                new_gps_data.append(0)
         
-    #     return new_gps_data
+        return new_gps_data
 
     def location_data(self, data:pd.Series) -> pd.Series:
```

Comment:
Now that we're not dealing with the `gps_location` column, we can actually average the data, as it's no longer stored as a string!